### ISSUE TITLE
server side metrics 

### ISSUE THREAD

**Issue by MohamedKHALILRouissi** [mlflow/mlflow#14149]
does mlflow expose server side metrics such  RETRY_ALL, RETRY_FAILED, RETRY_TIMED_OUT or RETRY_FAILED_AND_TIMED_OUT or any metrics related to mlflow backend not model metrics

**Comment by serena-ruan** [comment]
Not really. We support logging some system metrics though, you can use `mlflow.enable_system_metrics_logging()`
